### PR TITLE
Fix: Clean Notifications Code

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

Clean code.

# Changes

* Remove wrapper always present in Toasts.
* Use symbol instead of number for ToastMsg id.

# Tests

* Update Toasts Tests

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker please review

File: frontend/svelte/src/lib/components/ui/Toasts.svelte [hunk 1]
```diff
@@ -3,14 +3,13 @@
...
-<!-- We need the wrapper to avoid having an error in the tests -->
-<!-- TypeError: Cannot read properties of null (reading 'removeChild') -->
-<!-- https://github.com/sveltejs/svelte/issues/6037 -->
-<div class="wrapper">
-  {#each $toastsStore as msg (msg.id)}
-    <Toast {msg} />
-  {/each}
-</div>
+{#if $toastsStore.length > 0}
+  <div class="wrapper">
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/Toasts.svelte:7]: if issue is resolved, you still need the `<div class="wrapper">`?

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/ui/Toasts.svelte:7]: Yes, this one is needed to stack the toast notifications.

Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/ui/Toasts.svelte:7]: Thanks for the feedback

**Review by peterpeterparker — APPROVED** [review]

**Pull request merged**